doc/tutorial/first_steps.md

```diff
@@ -0,0 +1,290 @@
+It covers installing and initializing LXD, creating and configuring some instances, interacting with the instances, and creating snapshots.
+
+After going through these steps, you will have a general idea of how to use LXD, and you're ready to explore more advanced use cases!
+
+## Install and initialize LXD
+
+The easiest way to install LXD is to install the snap package.
+If you prefer a different installation method, or use a Linux distribution that is not supported by the snap package, see {ref}`installing`.
+
+1. Install `snapd`:
+
+   1. Run `snap version` to find out if snap is installed on your system:
+
+      ```{terminal}
+      :input: snap version
+
+      snap    2.59.4
+      snapd   2.59.4
+      series  16
+      ubuntu  22.04
+      kernel  5.15.0-73-generic
+      ```
+
+      If you see a table of version numbers, snap is installed and you can continue with the next step of installing LXD.
+
+   1. If the command returns an error, run the following commands to install the latest version of ``snapd``:
+
+          sudo apt update
+          sudo apt install snapd
+
+1. Enter the following command to install LXD:
+
+       sudo snap install lxd
+
+   If you get an error message that the snap is already installed, run the following command to refresh it:
+
+       sudo snap refresh lxd
+
+1. Enter the following command to initialise LXD:
+
+       lxd init --minimal
+
+   This will create a minimal setup with default options.
+   If you want to tune the initialization options, see {ref}`initialize` for more information.
+
+## Launch and inspect instances
+
+LXD is image based and can load images from different image servers.
+In this tutorial, we will use the official [`ubuntu:`](https://cloud-images.ubuntu.com/releases/) image server.
+
+You can list all images that are available on this server with:
+
+    lxc image list ubuntu:
+
+See {ref}`images` for more information about the images that LXD uses.
+
+Now, let's start by launching a few instances.
+With *instance*, we mean either a container or a virtual machine.
+See {ref}`containers-and-vms` for information about the difference between the two instance types.
+
+For managing instances, we use the LXD command line client `lxc`.
+See [About `lxd` and `lxc`](lxd-lxc) if you are confused about when to use the `lxc` command and when to use the `lxd` command.
+
+1. Launch a container called "first" using the Ubuntu 22.04 image:
+
+       lxc launch ubuntu:22.04 first
+
+   ```{note}
+   Launching this container takes a few seconds, because the image must be downloaded and unpacked first.
+   ```
+
+1. Launch a container called "second" using the same image:
+
+       lxc launch ubuntu:22.04 second
+
+   ```{note}
+   Launching this container is quicker than launching the first, because the image is already available.
+   ```
+
+1. Copy the first container into a container called "third":
+
+       lxc copy first third
+
+1. Launch a VM using the Ubuntu 22.04 image:
```

Comment:
```suggestion
1. Launch a VM called "vm" using the Ubuntu 22.04 image:
```

Just to be in line with the other examples above.

Reply:
To avoid any confusion, I'd go with something like `jammy-vm` or `ubuntu-vm`.